addons/sys_prc152/menus/fnc_changeMenu.sqf

```diff
@@ -22,15 +22,15 @@ params["_newMenu"];
 
 /*
 if(!([_newMenu] call FUNC(verifyIsMenu)) ) then {
-    _newMenu = GET_STATE_DEF("currentHome", GVAR(VULOSHOME));
+    _newMenu = GET_STATE_DEF(currentHome, GVAR(VULOSHOME));
 };
 */
 // clear menu data
-SET_STATE("menuSelection" , 0);
+SET_STATE(menuSelection , 0);
```

Comment:
Should be fixed by https://github.com/IDI-Systems/acre2/pull/48/commits/50c0de446a85dfa0d8d5fc99457ec22a2b752e47